crowbar_framework/app/models/backup.rb

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright 2015, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+class Backup
+  include ActiveModel::Model
+
+  attr_accessor :name, :created_at, :filename, :path
+
+  validates_presence_of :name, :created_at
+  validates :created_at, exclusion: { in: [nil, ""] }
+
+  def initialize(options)
```

Comment:
hmm strange it worked for me, but i guess you meant adding `options = {}`. okay will do